docs/release-notes/v0.39.0.md

```diff
@@ -0,0 +1,72 @@
+# Announcing Radius v0.39.0
+
+Today we're happy to announce the release of Radius v0.39.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+### Dapr Building Block: Configuration Store
+https://github.com/radius-project/radius/pull/7906
+
+### Add an in-memory data store
+https://github.com/radius-project/radius/pull/7950
+
+### Implement PostgreSQL datastore
+https://github.com/radius-project/radius/pull/7961
+
+## Breaking changes
+https://github.com/radius-project/radius/pull/7906
```

Comment:
I think `Implement PostgreSQL datastore` is too early of an addition to call out in highlights, we should call it out when it's available for users to switch to a postgres datastore.

I don't think this is a breaking change, so I removed it: https://github.com/radius-project/radius/pull/7906

```suggestion
## Highlights

### Support for Dapr Configuration Building Block
A new `Applications.Dapr/configurationStores` resource type is now available, which enables built-in support for the Dapr [Configuration](https://docs.dapr.io/getting-started/quickstarts/configuration-quickstart/) building block in Radius. The Dapr Configuration building block allows for dynamic configuration updates, including feature flag management, and is a great addition to the set of supported Dapr features in Radius. To learn more, go to the [Dapr Configuration Store resource schema](https://docs.radapp.io/reference/resource-schema/dapr-schema/configurationstore/) in the Radius documentation.

### In-memory data store for testing and development
Radius has added an implementation for an in-memory version of the data store interface, which may be used for testing and development purposes. This helps to speed up and simplify the existing ETC.d and mocking mechanisms that are frequently used today in tests.

## Breaking changes
None.
```